RELEASING.md

```diff
@@ -110,6 +110,16 @@ OpenSearch only does major releases when there are significant breaking changes
 
 For the actual steps to build a release, please see [Releasing OpenSearch](https://github.com/opensearch-project/opensearch-build/blob/main/README.md#releasing-opensearch).
 
+### Release Process for Changing Release Date
+
+In the OpenSearch project, we strive for consistent and predictable release schedule as multiple organizations and users depend on the software for their own projects and businesses.
```

Comment:
I think we need to link to the place that says how (often) OpenSearch releases.

Reply:
I think we should update https://opensearch.org/releases.html#release-schedule with these details as well.

Reply:
Can you point me to it? I will include it. Right now it's on OpenSearch.org, though it won't always be there.

Reply:
https://github.com/opensearch-project/project-website/blob/main/releases.md